data/language/en-GB.txt

```diff
@@ -3642,6 +3642,9 @@ STR_6383    :Open download page
 STR_6384    :Snow
 STR_6385    :Heavy Snow
 STR_6386    :Blizzard
+STR_6387    :Can't lower element here...
+STR_6388    :Can't raise element here...
+STR_6389    :No clearance
```

Comment:
This is not correct. The message gets emitted when the clearance height ends up with a value < 0 or > MAX_HEIGHT, while "No clearance" would mean that the tile element is too high to fit between/under another (which the tile element very deliberately ignores!)